dist/obs-api-testsuite-rspec.spec

```diff
@@ -52,7 +52,7 @@ popd
 
 pushd src/api
 # configure to the bundled gems
-bundle --local --path %_libdir/obs-api/
+bundle --local --path %_libdir/obs-api/ --without development test
```

Comment:
```suggestion
bundle --local --path %_libdir/obs-api/ --without development
```

I have not checked it, but most probably, to run the test suite we still need to build the gems of the environment test.

Reply:
Aaah, good catch... You're right